layouts/shortcodes/install-pulumi.html

```diff
@@ -35,13 +35,13 @@
 
 <div class="os-prologue-windows"></div>
 <div class="install_windows">
-    <p>Install Pulumi on Windows by running the installation script from a <code>cmd.exe</code> window:</p>
+    <p>Install Pulumi on Windows through the <a href="https://chocolatey.org" target="_blank">Chocolatey package manager</a>:</p>
     <div class="highlight">
-        <pre class="chroma"><code class="language-batch" data-lang="batch">@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; iex ((New-Object System.Net.WebClient).DownloadString('https://get.pulumi.com/install.ps1'))" && SET "PATH=%PATH%;%USERPROFILE%\.pulumi\bin"</code></pre>
+        <pre class="chroma"><code class="language-batch" data-lang="batch">choco install pulumi</code></pre>
     </div>
 </div>
 
 <p>
-    <i>For manual installation instructions, other platforms, or troubleshooting, see
+    <i>For alternative installation instructions (e.g. script-based installation, Windows installer, etc.) or troubleshooting, see
```

Comment:
Nit: We have a Windows installer?

Reply:
Yeah... I guess I shouldn't call it that. It's really just copy binaries from a .zip.